lib/get-config.js

```diff
@@ -25,11 +33,12 @@ module.exports = async (context, cliOptions) => {
   if (extendPaths) {
     // If `extends` is defined, load and merge each shareable config with `options`
     options = {
-      ...castArray(extendPaths).reduce((result, extendPath) => {
+      ...castArray(extendPaths).reduce(async(eventualResult, extendPath) => {
+        const result = await eventualResult;
         const extendsOptions = require(resolveFrom.silent(__dirname, extendPath) || resolveFrom(cwd, extendPath));
```

Comment:
switched back to loading config with `require` instead of `import` because of the need to load `.json` files for config, which isnt possible with `import` unless import assertions are used. since we are planning to raise the required node version to v18, import assertions should be available, but would require knowing the format of the config file before loading it. might be worth considering, but skipping for now